DigitalLearningSolutions.Web/Views/Register/Index.cshtml

```diff
@@ -0,0 +1,55 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Register
+@model RegisterViewModel
+
+@{
+  ViewData["Title"] = "Register";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Register</h1>
+
+    @if (Model.CentreName != null) {
+      <p class="nhsuk-body-l">
+        You have chosen to register for an account at <b>@Model.CentreName</b>.
+      </p>
+    }
+
+    <p class="nhsuk-body-l">
+      If you already have an account on DLS, we recommend that you log in before registering, in order to link your new account to your existing login.
+    </p>
+
+    <p class="nhsuk-body-l">
+      <a
+        class="nhsuk-action-link__link nhsuk-link--no-visited-state"
+        asp-controller="RegisterAtNewCentre"
+        asp-action="Index"
+        asp-route-centreId="@Model.CentreId"
+        asp-route-inviteId="@Model.InviteId">
+        <svg class="nhsuk-icon nhsuk-icon__arrow-right-circle" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+          <path d="M0 0h24v24H0z" fill="none"></path>
+          <path d="M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z"></path>
+        </svg>
+        <span class="nhsuk-action-link__text">Register with existing login</span>
+      </a>
+    </p>

```

Comment:
I think this (and the other link on the page) can be replaced by a `<vc:action-link />` component. You will have to put the query params in a `Dictionary<string, string>` and use the `asp-all-route-data` property of the component.

Reply:
Ah yeah...I thought I had seen that around before, but couldn't find it yesterday